src/commands/general/msfs.ts

```diff
@@ -0,0 +1,20 @@
+import Discord from 'discord.js';
+import { CommandCategories, CommandDefinition } from '../index';
+import { color } from '../../index';
+
+export const msfs: CommandDefinition = {
+    names: ['msfs'],
+    description: "Redirects user to MSFS server for support",
+    category: CommandCategories.GENERAL,
+    execute: async (message, args) => {
+        const embed = new Discord.MessageEmbed()
+            .setColor(color)
+            .setTitle('Sim Issue')
+            .setDescription(
+                'This is an issue regarding Microsoft Flight Simulator, as this is the Digital Flight Dynamics A350X Project server, we are unable to provide support for the same.'+
```

Comment:
`This is an issue regarding Microsoft Flight Simulator` should be its own sentence.